### PR TITLE
storage: add swap example

### PR DESCRIPTION
**Pull request by bgilbert**
Requires FCCT 0.10.0.

From https://coreos.github.io/fcct/examples/.

### CODE REVIEW

**Comment by JaimeMagiera** [comment]
Note that #239, which is awaiting review, deletes ign-storage.adoc.

**Comment by bgilbert** [comment]
Cool, I think that's consistent with what this PR is doing.

File: modules/ROOT/pages/storage.adoc [hunk 1]
```diff
@@ -233,6 +233,29 @@ storage:
...
 
+== Adding swap
+
+This example creates a swap partition spanning all of the `sdb` device, creates a swap area on it, and creates a systemd swap unit so the swap area is enabled on boot.
+
+.Configuring a swap partition on a second disk
+[source,yaml]
+----
+variant: fcos
+version: 1.4.0-experimental
```
Comment (cgwalters) [inline modules/ROOT/pages/storage.adoc:244]: Hmm but should we really be documenting using the experimental fcc version?

**Review by cgwalters — COMMENTED** [review]
Hmm

File: modules/ROOT/pages/storage.adoc (hunk 1 above)
Comment (bgilbert) [reply modules/ROOT/pages/storage.adoc:244]: Hmm, it's a good point.  We have done it once before, in #172.  I'm okay with adding a `NOTE` or with deferring the PR outright until whenever 1.4.0 stabilizes.  Thoughts?

Comment (arithx) [reply modules/ROOT/pages/storage.adoc:244]: IMO it's situational, for something like LUKS I think getting the documentation in early (with maybe an issue being created to track the dropping of `-experimental` once the spec lands). I'd also be fine if we just want to defer until 1.4.0 stabilizes.

Comment (jlebon) [reply modules/ROOT/pages/storage.adoc:244]: No strong opinion either way myself.

Comment (bgilbert) [reply modules/ROOT/pages/storage.adoc:246]: I've added a NOTE, on the theory that it's better to document experimental FCOS functionality with caveats than not document it at all.

**Review by jlebon — APPROVED** [review]

**Pull request merged**